### PR TITLE
Flight interface tests

### PR DESCRIPTION
**Pull request by Xierumeng**
Also fixed file naming.

### CODE REVIEW

File: tests/test_flight_interface_worker.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import queue
+import time
+
+from modules import odometry_and_time
+from modules.flight_interface import flight_interface_worker
+from utilities.workers import queue_proxy_wrapper
+from utilities.workers import worker_controller
+
+
+MAVLINK_ADDRESS = "tcp:localhost:14550"
```
Comment (mgupta27) [inline tests/test_flight_interface_worker.py:14]: Nit: Rename to `CONNECTION_ADDRESS`

**Review by mgupta27 — COMMENTED** [review]
Reviewed.

File: tests/test_flight_interface_worker.py (hunk 1 above)
Comment (Xierumeng) [reply tests/test_flight_interface_worker.py:14]: `MAVLINK_CONNECTION_ADDRESS` ?

**Review by mgupta27 — APPROVED** [review]
LGTM!

**Pull request merged**